### DISCUSSION TITLE
Questions about G-helper as a newbie

### DISCUSSION THREAD

**Discussion by gunl1201** [seerge/g-helper#3036] [Q&A]
Hello, I'm new to this git hub thing, and I recently bought a Zephyrus g16 2024 with 4090 and ultra 9 185h. I have a few questions, and I'm sorry if this is a common question asked or just a dumb question.

1. I do not have the advanced tab at all in G-helper fans and power. Is that supposed to be the case?
2. Should the GPU and advanced tab appear only when using standard or ultimate mode in G-helper fans and power?
3. I followed a YouTube video for installing G-helper, but on the CPU page, I have CPU pl 1 and pl 2, but in the video, it had spl, spot, and ppt. Is it because it was an old version?
4. Is vbios worth updating?
5. Are there any tips on keeping the battery healthy and the OLED screen from burn in?

Thanks

**Comment by seerge** [comment]
@gunl1201 hello

1. Advanced tab contains unervolting options for AMD devices, you have Intel so you don't supposed to have it
2. GPU tab appears only when GPU is enabled (i.e. you are not in `Eco` mode)
3. You generally don't need to set any custom limits or fan curves unless you pursue some special goal. PL1 and PL2 are Intel equivalents of SPL and SPPT correspondingly, both of them can set maximum power your CPU is allowed to consume.
4. If it's official updates from the manufacturer - it's always better to keep your device up to date
5. Keeping battery limit on 80% for most of the time is usually enough. For OLED just use common sense, and don't max out brightness or keep same static elements on the same spots all the time and set windows to turn off display quickly when laptop is inactive.

**Reply by gunl1201** [reply]
Sorry instead of vbios updating I meant vbios flashing. Is it worth doing it?
